tests/publiccloud/ssh_end.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: This test will leave the SSH interactive session, kill the SSH tunnel and destroy the public cloud instance
+#
+# Maintainer: Pavel Dostal <pdostal@suse.cz>
+
+use Mojo::Base 'publiccloud::basetest';
+use warnings;
+use testapi;
+use strict;
+use utils;
+
+sub run {
+    my ($self, $args) = @_;
+    select_console 'root-console';
+
+    # Leave the SSH interactive session
+    type_string("exit\n");
```

Comment:
Actually I was talking about line 25. It is just a sanity check, e.g. the ssh connection get's disconnected during the test, we would fail afterwards or?

Reply:
Ah I see, that's a good idea! Maybe we can just check if the `/dev/sshserial` exists and if it is a FIFO file :smile: ?

Reply:
actually valid. Lets go for it!